mrcnn/model.py

```diff
@@ -2261,7 +2260,7 @@ def set_log_dir(self, model_path=None):
                                         int(m.group(4)), int(m.group(5)))
                 # Epoch number in file is 1-based, and in Keras code it's 0-based.
                 # So, adjust for that then increment by one to start from the next epoch
-                self.epoch = int(m.group(6)) - 1 + 1
```

Comment:
The `-1 + 1` is intentional to clarify the comment above it. To make it clear that we're subtracting 1 then adding 1.